src/nightly.js

```diff
@@ -128,7 +128,7 @@ module.exports = function*(argv) {
     yield npmpublish.publishTag(options);
 
     //unpublish old nightly
-    options.version = oldNightlyVersion;
+    options.version = oldNightlyVersions;
     yield npmpublish.unpublish(options);
```

Comment:
`src/npm-publish.js` will be needed to be updated in this case too as it was initially handling a single version.